src/ui/components/results/verticalresultscomponent.js

```diff
@@ -155,8 +163,15 @@ export default class VerticalResultsComponent extends Component {
     /**
      * Config options used in the {@link ResultsHeaderComponent}
      */
-    const { showFieldNames, resultsCountSeparator, showResultCount, showAppliedFilters, showChangeFilters } = this._config;
-    this.resultsHeaderOpts = { showFieldNames, resultsCountSeparator, showResultCount, showAppliedFilters, showChangeFilters };
+    this.resultsHeaderOpts = {
+      showFieldNames: this._config.appliedFilters.showFieldNames,
+      resultsCountSeparator: this._config.appliedFilters.resultsCountSeparator,
+      showAppliedFilters: this._config.appliedFilters.show,
+      showChangeFilters: this._config.showChangeFilters,
+      showResultCount: this._config.showResultCount,
+      removable: this._config.appliedFilters.removable, // TODO implement
+      delimiter: this._config.appliedFilters.delimiter || '|'
```

Comment:
We should include this default in the VerticalResultsConfig class as well. It should provide a fully populated appliedFilters config (using any defaults necessary to do this). The component should not need any additional defaulting behavior for _config.appliedFilters.

Reply:
sorry brainfartedI also missed delimiter and removable in the config class